src/main/java/org/aleksey/springnew/service/TodoService.java

```diff
@@ -0,0 +1,18 @@
+package org.aleksey.springnew.service;
+
+import org.aleksey.springnew.dto.TodoCreateDto;
+import org.aleksey.springnew.dto.TodoHistoryResponseDto;
+import org.aleksey.springnew.dto.TodoResponseDto;
+import org.aleksey.springnew.dto.TodoUpdateDto;
+
+import java.util.List;
+
+public interface TodoService {
+    TodoResponseDto createTodo(TodoCreateDto todoCreateDto);
+
+    TodoResponseDto updateTodo(Long id, TodoUpdateDto todoUpdateDto);
+
+    void deleteTodo(Long id);
+
+    List<TodoHistoryResponseDto> getTodoHistory(Long id);
+}
```

Comment:
You can skip `todo` for simplicity, because you have these methods in TodoService, so you have a bounded context.
```suggestion
public interface TodoService {
    TodoResponseDto create(TodoCreateDto todoCreateDto);

    TodoResponseDto update(Long id, TodoUpdateDto todoUpdateDto);

    void deleteById(Long id);

    List<TodoHistoryResponseDto> getHistory(Long id);
}
```